### ISSUE TITLE
Log Level is changed with reload, without reload it doesn't work.

### ISSUE THREAD

**Issue by nedad** [tony19/logback-android#182]
Hello,

I am using your com.github.tony19:logback-android:1.1.1-12 library.

After I change my Log Level in Property file, I expect that the Log Level is changed in runtime. But ist doesn't work.

Do you know what could be  a Problem?

I have set scan=true and scanPeriod=2 seconds. I have every time after I set the change in properties file to restart the Android App. Because only in this way works.

logback.properties

```
# Properties in this file act as fallback values. They get overridden by specific ones dependent on the chosen flavor
root.loglevel=OFF
```

logback.xml

```xml
<configuration debug="true" scan="true" scanPeriod="2 seconds">

    <include file="${EXT_DIR}/Download/de.hermes.alpha/assets/logbackincludedfile.xml"
        optional="true" />
    <root level="${root.loglevel:-DEBUG}">
        <appender-ref ref="logcat" />
        <appender-ref ref="logstash" />
        <appender-ref ref="rollingLogFile" />
    </root>

<?xml version="1.0"?>

<included>

<property value="OFF" name="root.level"/>

</included>
```

Regards,
Neda

**Comment by tony19** [comment]
### Bug in file-change detection

There's a bug in `1.1.1-12` that prevents `<included>` file-change detection from occurring, which blocks the `scan` feature you're seeking. Specifically, file-change detection is disabled if the main configuration URL (to the JAR resource of `assets/logback.xml`) cannot be determined, which is occurring because `ContextInitializer` is passing an `InputStream` to [`GenericConfigurator.doConfigure(InputStream)`](https://github.com/tony19/logback-android/blob/903d4a672ca35e1e179a1c6ba85fccafcb25513a/logback-android/src/main/java/ch/qos/logback/core/joran/GenericConfigurator.java#L116), which does not set the main configuration URL, required for config-file change detection to proceed.

This will be fixed in the next release.

### Only `included` files are scanned 

Your comment below makes an incorrect assumption that files specified in <property> are scanned:

> After I change my Log Level in Property file, I expect that the Log Level is changed in runtime.

The `.properties` file (included via <property file="...">) does not get scanned. In this case, only files from `<include file="...">` are scanned periodically when `<configuration scan="true">`.

**Comment by tony19** [comment]
Fixed in [1.1.1-13](https://github.com/tony19/logback-android/releases/tag/v_1.1.1-13)
